data_dir/c-s_lwt_basic.yaml

```diff
@@ -65,8 +72,8 @@ queries:
       cql: select * from blogposts where domain = ? LIMIT 1
       fields: samerow
    lwt_update_one_column:
-      cql: update blogposts set lwt_ind = 10000001 where domain = ? and published_date = ? if lwt_ind < 0
+      cql: update blogposts set lwt_ind = 30000000 where domain = ? and published_date = ? if lwt_ind < 0
       fields: samerow
    lwt_update_two_columns:
```

Comment:
two conditions? (I see 3 conditions, so not sure)

Reply:
As it's written: update two columns